yarn.lock

```diff
@@ -116,6 +116,28 @@
     invariant "^2.2.4"
     semver "^5.5.0"
 
+"@babel/core@7.10.5":
```

Comment:
Are all of these yarn.lock changes intended? Or do you need to update dependency versions in package.json?